src/compiler/transformers/classFields.ts

```diff
@@ -7,6 +7,23 @@ namespace ts {
          */

```

Comment:
@RyanCavanaugh @DanielRosenwasser @robpalme @joeywatts  cc Ron Nathan and Ron
Do we want to ban the following features (in the checker) for now?
- private-named statics
- private-named methods

If so, could you advise on how best to do the ban?

They parse and check fine, but we don't have emit yet. We have started the emit work in another branch, but had agreed to land them in follow-on PRs.

Reply:
I think we should error on them during parse for now and remove the errors when they are supported.

Reply:
consolidating this discussion into: https://github.com/microsoft/TypeScript/pull/30829#discussion_r306051459